### PR TITLE
Cypress/lnurl login

### LINKED ISSUE #173: [Cypress] Log-in with LNURL
Use cypress to log-in to the bounties site with LNURL.

### PR DESCRIPTION
**Pull request by tobi-bams**
## Issue ticket number and link
Closes #173

## Type of change

Please delete options that are not relevant.

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by tobi-bams** [comment]
@ecurrencyhodler here is PR for login with LNURL AUTH and attached is a video

https://github.com/stakwork/sphinx-tribes-frontend/assets/56609046/94cef704-929a-49e0-8de4-c29ab2cf0b9f

**Comment by ecurrencyhodler** [comment]
YOU ARE THE MAN TOBI!!! 🔥🔥🔥🔥

**Comment by tobi-bams** [comment]
> YOU ARE THE MAN TOBI!!! 🔥🔥🔥🔥

Thank you!!!

File: src/config/socket.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { getHost } from './host';
 export const URL =
   process.env.NODE_ENV !== 'development'
     ? `wss://${getHost()}/websocket`
-    : `ws://127.0.0.1:5005/websocket`;
+    : `ws://${getHost()}/websocket`;
```
Comment (elraphty) [inline src/config/socket.ts:6]: @tobi-bams Did you test if `localhost:5005 `works with the WebSocket connection?

Comment (tobi-bams) [reply src/config/socket.ts:6]: Yes, I tested and it works

**Comment by ecurrencyhodler** [comment]
Paid!

**Pull request merged**